Mage.Sets/src/mage/cards/m/MaskwoodNexus.java

```diff
@@ -81,31 +82,21 @@ public boolean apply(Game game, Ability source) {
         }
         // Creature cards you own that aren't on the battlefield
         // in graveyard
-        for (UUID cardId : controller.getGraveyard()) {
-            Card card = game.getCard(cardId);
-            if (card != null && card.isCreature(game)) {
-                game.getState().getCreateMageObjectAttribute(card, game).getSubtype().setIsAllCreatureTypes(true);
-            }
-        }
+        Set<Card> affectedCards =
+                controller.getGraveyard().stream().map(game::getCard).filter(Objects::nonNull).filter(card -> card.isOwnedBy(controller.getId())).collect(Collectors.toSet());
```

Comment:
style, use new line for each method call when processing a stream

Also, only exile zone needs the owner check, graveyard/hand/library will always be owned

Reply:
I believe exile is checking the owner?